libraries/deeplink/impl/src/test/kotlin/io/element/android/libraries/deeplink/impl/DefaultDeepLinkCreatorTest.kt

```diff
@@ -19,15 +24,43 @@ class DefaultDeepLinkCreatorTest {
     @Test
     fun create() {
         val sut = DefaultDeepLinkCreator()
-        assertThat(sut.create(A_SESSION_ID, null, null, null))
-            .isEqualTo("elementx://open/@alice:server.org")
-        assertThat(sut.create(A_SESSION_ID, A_ROOM_ID, null, null))
-            .isEqualTo("elementx://open/@alice:server.org/!aRoomId:domain")
-        assertThat(sut.create(A_SESSION_ID, A_ROOM_ID, A_THREAD_ID, null))
-            .isEqualTo("elementx://open/@alice:server.org/!aRoomId:domain/\$aThreadId")
-        assertThat(sut.create(A_SESSION_ID, A_ROOM_ID, A_THREAD_ID, AN_EVENT_ID))
-            .isEqualTo("elementx://open/@alice:server.org/!aRoomId:domain/\$aThreadId/\$anEventId")
-        assertThat(sut.create(A_SESSION_ID, A_ROOM_ID, null, AN_EVENT_ID))
-            .isEqualTo("elementx://open/@alice:server.org/!aRoomId:domain//\$anEventId")
+        val sessionId = A_SESSION_ID
+        val roomId = A_ROOM_ID
+        val threadId = A_THREAD_ID
+        val eventId = AN_EVENT_ID
+        assertThat(sut.create(sessionId, null, null, null))
+            .isEqualTo("elementx://open/${sessionId.urlEncoded()}")
```

Comment:
I think it's better to have raw string in the expectation, rather than invoking `sessionId.urlEncoded()`. If there is an issue in `urlEncoded`, like for instance if it's noop, the test will not detect it.